ui/next/app/redux/uiData.ts

```diff
@@ -153,16 +153,22 @@ export function loadUIData(...keys: string[]) {
     dispatch(fetchUIData());
 
     return getUIData({ keys }).then((response) => {
-      response.getKeyValues().forEach((val, key) => {
+      let keyValues = response.getKeyValues();
+
+      _.each(keys, (key) => {
```

Comment:
Does this compile?  getKeyValues() returns a ProtobufMap, I don't think it can be iterated using underscore.